src/util/apiUsageKey.ts

```diff
@@ -0,0 +1,15 @@
+import { APIGatewayProxyEventHeaders } from 'aws-lambda';
+
+const usageTokenRegex = new RegExp(/^.{10,}$/g);
+
+const validUsageKey = (headers:APIGatewayProxyEventHeaders):boolean => {
+  if (headers['X-Api-Key']) {
+    if (usageTokenRegex.test(headers['X-Api-Key'])) {
+      return true;
+    }
+    return false;
+  }
+  return false;
```

Comment:
@JustinJohnston-alt  - this function could be simplified

```suggestion
  return headers['X-Api-Key'] && usageTokenRegex.test(headers['X-Api-Key']);
```